features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/Publisher.jsx

```diff
@@ -155,21 +154,20 @@ class Publisher extends React.Component {
      */
     render() {
         const { user, userResolved } = this.state;
-        const { pathname } = window.location;
-        const params = qs.stringify({
-            referrer: pathname.split('/').reduce((acc, cv, ci) => (ci <= 1 ? '' : acc + '/' + cv)),
-        });
+
         if (!userResolved) {
             return <Progress />;
         }
+        if (!user) {
+            window.location = '/publisher-new/services/auth/login';
```

Comment:
Should we take publisher-new from the configuration js?

Reply:
@malinthaprasan  , I will fix them